docs/adaptors-storm.md

```diff
@@ -10,7 +11,7 @@ An application can inject data into a Storm topology via a generic Pulsar spout,
 
 ## Using the Pulsar Storm Adaptor
 
-Include dependency for Pulsar Storm Adaptor:
+To use the Pulsar Storm Adaptor, you need to include dependency for Pulsar Storm Adaptor:
 
 ```xml
 <dependency>
```

Comment:
Out of thread, @cbornet is the GAV of pulsar storm adaptor changed?

Reply:
What is "GAV" ?

Reply:
group id, artifact id, version; that is, a unique identifier for a dependency.

Reply:
Oh ok. The storm adapter has moved to the Storm project so we should update the doc.